src/main/java/org/jfrog/hudson/util/ExtractorUtils.java

```diff
@@ -476,6 +476,11 @@ public static void persistConfiguration(ArtifactoryClientConfiguration configura
                 Properties properties = new Properties();
                 properties.putAll(configuration.getAllRootConfig());
                 properties.putAll(configuration.getAllProperties());
+                // Properties that have the 'artifactory.' prefix are deprecated.
+                // For backward compatibility reasons, both will be added to the props map.
```

Comment:
Let's also add to this description the minimum build-info version which doesn't use these deprecated properties anymore. This will make it easier for us to decide when in the future we will stop supporting these properties.

Reply:
We don't know what is the Gradle extractor's version since it can be changed in the build.gradle file.